clippy.toml

```diff
@@ -0,0 +1 @@
+upper-case-acronyms-aggressive = true
```

Comment:
This is a clippy option for `upper_case_acronyms`:

> Triggers if there is more than one uppercase char next to each other

So it catches things like `CamelCASE`.

Reply:
How does `RandomXVm` pass this lint, the `XV` are uppercase characters next to each other.

Reply:
[There's special conditions for acronyms](https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/src/upper_case_acronyms.rs#L59-L72), either:

- 3 uppercase chars back-to-back
- 2 ending uppercase chars

In `RandomXVm`'s case, the last `m` resets the uppercase char count to 0...